deploy/ocp-templates/glusters3template.json

```diff
@@ -0,0 +1,205 @@
+                                "mountPath": "/sys/fs/cgroup"
+                            }
+                        ],
+                        "terminationMessagePath": "/dev/termination-log",
+                        "imagePullPolicy": "IfNotPresent",
+                        "securityContext": {
+                            "privileged": true
+                        }
+                    }
+                ],
+                "restartPolicy": "Always",
+                "terminationGracePeriodSeconds": 30,
+                "dnsPolicy": "ClusterFirst",
+                "serviceAccountName": "default",
+                "serviceAccount": "default",
+                "securityContext": {}
+            },
+            "status": {}
+        },
+        {
+            "kind": "Service",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "glusters3service",
+                "creationTimestamp": null,
+                "labels": {
+                    "name": "glusters3"
+                }
+            },
+            "spec": {
+                "ports": [
+                    {
+                        "protocol": "TCP",
+                        "port": 8080,
+                        "targetPort": 8080
+                    }
+                ],
+                "selector": {
+                    "name": "glusters3"
+                },
+                "type": "ClusterIP",
+                "sessionAffinity": "None"
+            },
+            "status": {
+                "loadBalancer": {}
+            }
+        },
+        {
+            "kind": "PersistentVolumeClaim",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "glusterfs-claim",
+                "creationTimestamp": null,
+                "annotations": {
+                    "volume.beta.kubernetes.io/storage-class": "s3storageclass"
+                }
+            },
+            "spec": {
+                "accessModes": [
+                    "ReadWriteMany"
+                ],
+                "resources": {
+                    "requests": {
+                        "storage": "${VOLUME_CAPACITY}"
+                    }
+                }
+            }
+        },
+        {
+            "kind": "PersistentVolumeClaim",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "glusterfs-claim-meta",
+                "creationTimestamp": null,
+                "annotations": {
+                    "volume.beta.kubernetes.io/storage-class": "s3storageclass"
+                }
+            },
+            "spec": {
+                "accessModes": [
+                    "ReadWriteMany"
+                ],
+                "resources": {
+                    "requests": {
+                        "storage": "1Gi"
+                    }
+                }
+            }
+        }
+    ],
+  "parameters": [
+    {
+      "name": "GLUSTER_VOLUMES",
```

Comment:
We need a string to be passed to gluster-s3 container for swift-genbuilders and for mounting at specific location.